packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Mocks/MockHelpers.cs

```diff
@@ -16,14 +16,16 @@ namespace Microsoft.Generator.CSharp.ClientModel.Tests
     internal static class MockHelpers
     {
         public const string MocksFolder = "Mocks";
+        public static string ConfigFilePath = Path.Combine(AppContext.BaseDirectory, MocksFolder);
 
         public static void LoadMockPlugin(
             Func<InputType, IReadOnlyList<TypeProvider>>? createSerializationsCore = null,
             Func<InputType, CSharpType>? createCSharpTypeCore = null,
             Func<CSharpType>? matchConditionsType = null,
             Func<CSharpType>? tokenCredentialType = null,
             Func<InputOperation, TypeProvider, MethodProviderCollection>? createMethods = null,
-            Func<InputParameter, ParameterProvider>? createParameter = null)
+            Func<InputParameter, ParameterProvider>? createParameter = null,
+            InputLibrary? inputLibrary = null)
```

Comment:
Lets use Func<InputLibrary>? as the type so we can handle more cases if people want to pass in something more complex